lib/teleterm/clusters/cluster_databases.go

```diff
@@ -245,3 +246,15 @@ type GetDatabasesResponse struct {
 	// // TotalCount is the total number of resources available as a whole.
 	TotalCount int
 }
+
+// NewcmdCLICmdBuilder creates a dbcmd.MakeDbcmdCLICmdBuilder with provided
+// cluster, db route, and options.
+func NewDbcmdCLICmdBuilder(cluster *Cluster, routeToDb tlsca.RouteToDatabase, options ...dbcmd.ConnectCommandFunc) *dbcmd.CLICommandBuilder {
```

Comment:
The `NewDbcmdCLICmdBuilder` is a bit confusing: 
- `cmd` keyword is repeated: New...cmd...Cmd..